doc-Managing_Infrastructure_and_Inventory/topics/Adding_a_Cloud_Network.adoc

```diff
@@ -0,0 +1,20 @@
+[[adding-a-cloud-network]]
+
+=== Adding a Cloud Network
+
+
+Add a new cloud network following the steps in this procedure:
+
+. Navigate to menu:Networks > Networks. 
+. Click image:1847.png(Configuration) and click *Add a new Cloud Network*.
```

Comment:
The navigation menu and the image syntax needs an edit for it to render correctly.